src/components/Letters.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component} from 'react'
+import Letter from './Letter'
+
+class Letters extends Component{
+
+    generateLetterWord(letters){
+        const lettersArray = []
+        for(let i in letters){
+            // console.log(this.props.letterStatus[i]);
+            if(this.props.letterStatus[i]){
+                lettersArray.push(<Letter letter={i} key={i} className='selected'  selectLetter={this.props.selectLetter} />)
+            }else{
+                lettersArray.push(<Letter letter={i} key={i} className='non-selected'  selectLetter={this.props.selectLetter} />)
+            }
```

Comment:
instead of the if-else, you could do a ternary inside the className property like this:
`lettersArray.push(<Letter letter={i} key={i} className={this.props.letterStatus[i] ? 'selected' : 'non-selected'}  selectLetter={this.props.selectLetter} />)`